src/wallet/rpcwallet.cpp

```diff
@@ -31,7 +31,14 @@
 
 CWallet *GetWalletForJSONRPCRequest(const JSONRPCRequest& request)
 {
-    // TODO: Some way to access secondary wallets
+    if (!request.wallet.empty()) {
+        for (const auto& wallet : ::vpwallets) {
```

Comment:
Because we may want to add dynamic loading/unloading/creation of wallets in future. In fact, I'm working on a branch that does just that.